.babelrc

```diff
@@ -1,18 +1,18 @@
 {
-    "presets": [
-        [
-            "@babel/preset-env",
-            {
-                "targets": {
-                    // browsers released in 2021
-                    "chrome": "88",
-                    "firefox": "85",
-                    "safari": "14",
-                    // NodeJS 14
-                    "node": "14"
-                },
-                "loose": true
-            }
-        ]
-    ]
+	"presets": [
```

Comment:
I think we could actually skip babel and minification entirely in v2. If folks need broader browser support than what we provide or to minify, they can do their own transformations.